dev_guide/builds.adoc

```diff
@@ -846,43 +850,47 @@ source:
 
 == Binary Source
 
-When the `*BuildConfig.spec.source.type*` is `*Binary*`, the build will expect a
-binary as input, and an inline Dockerfile is optional.
-
-The binary is generally assumed to be a tar, gzipped tar, or zip file depending
-on the strategy. For `*Docker*` builds, this is the build context and an
-optional Dockerfile may be specified to override any Dockerfile in the build
-context. For `*Source*` builds, this is assumed to be an archive as described
-above. For `*Source*` and `*Docker*` builds, if `*binary.asFile*` is set the
-build context will consist of a single file named by the value of `*binary.asFile*`. 
-The `*contextDir*` field may be used when an archive is provided. Custom builds will 
-receive this binary as input on standard input (`stdin`).
+`*Binary*` is a valid value for `*BuildConfig.spec.source.type*`, but is it not something you will
```

Comment:
Yeah, as we talked about in scrum on Friday, Binary builds are a bit funky, and certainly that funkiness is more apparent here than perhaps in other parts of this description.  This type is unique in that it is truly optional in the need to specify it.  But let's see if I can convey that more clearly.

I'm not sure the suggestion necessarily works either, but I think I have a thought on how to combine the first two sentences.  I'll push that shortly, and let me know if it seems clearer to you.

thanks.